lib/pipeline.js

```diff
@@ -1147,6 +1149,59 @@ class PipelineModel extends BaseModel {
             .then(() => removeChildPipelines()) // remove pr events
             .then(() => super.remove()); // remove pipeline
     }
+
+    /**
+     * getMetrics for this pipeline
+     * @method getMetrics
+     * @param  {Object}   [config]              Configuration object
+     * @param  {String}   [config.startTime]    Look at events created after this startTime
+     * @param  {String}   [config.endTime]      Look at events created before this endTime
+     * @return {Promise}  Resolves to array of metrics for events belong to this pipeline
+     */
+    async getMetrics(config) {
```

Comment:
Can probably change to 
```suggestion
    async getMetrics({ startTime, endTime }) {
```
That way the next couple lines can be:
```
const startTime = startTime || null;
const endTime = endTime || null;
```

Reply:
Will get `startTime already declared` if do it that way.